sequencer/src/network/cdn.rs

```diff
@@ -43,22 +44,39 @@ impl<T: SignatureKey> SignatureScheme for WrappedSignatureKey<T> {
     type PublicKey = Self;
 
     /// Sign a message of arbitrary data and return the serialized signature
-    fn sign(private_key: &Self::PrivateKey, message: &[u8]) -> anyhow::Result<Vec<u8>> {
-        let signature = T::sign(private_key, message)?;
-        // TODO: replace with rigorously defined serialization scheme...
-        // why did we not make `PureAssembledSignatureType` be `CanonicalSerialize + CanonicalDeserialize`?
+    fn sign(
+        private_key: &Self::PrivateKey,
+        namespace: &str,
```

Comment:
Since we have "namepaces" elsewhere in this codebase might be good to have a type or some documentation what this namespace is used for.